docs/quantize.md

```diff
@@ -0,0 +1,96 @@
+# How to Quantize a Model
+
+Sometimes the model you want to work with is not available at [https://ollama.ai/library](https://ollama.ai/library). If you want to try out that model before we have a chance to quantize it, you can use this process.
```

Comment:
Remove `If you want to try out that model before we have a chance to quantize it, you can use this process.`